src/main/java/roomescape/controller/ReservationController.java

```diff
@@ -0,0 +1,45 @@
+package roomescape.controller;
+
+import roomescape.domain.Reservation;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import roomescape.dto.ReservationRequest;
+import roomescape.service.ReservationService;
+
+import java.net.URI;
+
+@RestController
+public class ReservationController {
+
+    private ReservationService reservationService;
```

Comment:
final 키워드도 함께 사용해주시면 더 좋을 것 같아요 :)

```suggestion
    private final ReservationService reservationService;
```